sync.go

```diff
@@ -45,6 +45,7 @@ func (s *SyncAction) Execute(username, password, override, vaultpass string) err
 		log.Info("- %s", file)
 	}
 
+	cli.Println("Processing...")
```

Comment:
That does not look like the best place. I think it makes more sense to be placed inbetween
`err := s.prepareArtifact`
and 
`modifiedFiles`